Cargo.toml

```diff
@@ -19,3 +19,12 @@ exclude = [
   "benches/payout-stakers",
   "bin/cliain",
 ]
+
+[profile.release]
+# Substrate runtime requires unwinding.
+panic = "unwind"
```

Comment:
If we go with the solution above, we don't need to specify this setting explicitly, as it will be present in the default `release` profile

Reply:
I was wrong , this just extends profile